repository/users.js

```diff
@@ -35,6 +35,24 @@ class UserRepository {
       },
     });
   }
+
+  getUserExpressions(userId) {
```

Comment:
Here you are getting expressions and not an user so isn't the right place for this is in the expressions repo? You can move that in the expression repo and with only one join with `UserExpression` you can get all expressions from that user.